docs/appendices/release-notes/unreleased.rst

```diff
@@ -69,14 +69,18 @@ None
 Changes
 =======
 
+- Add the :ref:`CHECK <check_constraint>` constraint syntax, which specifies
```

Comment:
I would not do this / thing, I don't think it adds clarity. A general plural ("values", "columns") is fine.